history/characters/10000_orc.txt

```diff
@@ -419,7 +420,8 @@
 	martial=6 diplomacy=5 stewardship=4 intrigue=6 learning=6
 	trait=education_learning_3 
 	trait=sadistic trait=wrathful
-	536.1.1={ birth=yes trait=creature_orc }
+	536.1.1={ birth=yes trait=creature_orc
+			effect = { make_important_lore_character_effect = yes } }
```

Comment:
Tab rules. The closing bracket must be on the same tab level as the opening bracket.
```suggestion
	536.1.1={
		birth=yes trait=creature_orc
		effect = { make_important_lore_character_effect = yes }
	}
```

Reply:
Should be good now. I also standardised the formatting for some pre-existing effect markers.